src/Sulu/Bundle/ActivityBundle/UserInterface/Controller/ActivityController.php

```diff
@@ -331,10 +331,19 @@ private function getTranslationParameters(array $array, string $prefix = '{', st
             } elseif (\is_numeric($value)) {
                 $translationParameters[$prefix . $key . $suffix] = $value;
             } elseif (\is_scalar($value)) {
-                $translationParameters[$prefix . $key . $suffix] = (string) $value;
+                $value = (string) $value;
+
+                if (\array_key_exists($key . 'Locale', $array)) {
+                    $valueLocale = $array[$key . 'Locale'] ?? null;
+                    $valueLocale = $valueLocale ? (string) $valueLocale : null;
+
+                    $value = $this->getLocalizedValue($value, $valueLocale, $translationLocale);
+                }
+
+                $translationParameters[$prefix . $key . $suffix] = $value;
             } elseif (\is_array($value)) {
-                foreach ($this->getTranslationParameters($value, '', '') as $translationParameterKey => $translationParameterValue) {
-                    $translationParameters[$prefix . $key . '.' . $translationParameterKey . $suffix] = $translationParameterValue;
+                foreach ($this->getTranslationParameters($value, $translationLocale, '', '') as $translationParameterKey => $translationParameterValue) {
+                    $translationParameters[$prefix . $key . '_' . $translationParameterKey . $suffix] = $translationParameterValue;
```

Comment:
@nnatter Just realized, that the js library `intl-messageformat` can't handle translations containing something like `{context.foo}` ... An error is thrown in the javascript console then. Therefore I changed it back to `{context_foo}`